### DISCUSSION TITLE
民告官自古无胜算

### DISCUSSION THREAD

**Discussion by zongjiangU** [WeNeedHome/SummaryOfLoanSuspension#574] [Ideas]
民告官者，杖五十，
触天威者，诛九族。
被欺压的百姓走投无路，
破釜沉舟的发个连状告书都算不上的声明，
围观群众中支持当事者的，在窃窃私语着，
吼天喊地的声音是在斥责着他人不懂规矩，不懂感恩，
更有甚者跑至衙门，
告发张三李四聚众闹事，
问其告发动机，不为碎银几两，只为心中正义。

自何时起，连说话都要看官家脸色？
又在何时，在这片土地上说话真的不必看官家脸色？

不要扯高丽国，美丽国怎样怎样的，
站在这片的人，对于他国月亮圆不圆，谁会关心？

我就想问，这片土地到底没有被解放？

**Comment by zxcs001** [comment]
随意，最好把我抓进去，关个一两年。

然后我扭头申请美国美国政治避难，在美国搞码农多香……

你用这种东西威胁一个已经不在乎征信，不在乎那些乱七八糟东西的人，有意义么？

共产党那套东西，只能威胁一下，收入不错，有体面生活的体面人。真的遇见烂尾，背上几十年贷款拿不到房，还要租房过的连狗的不如的人。

谁搭理你那些东西啊，懂了么，五毛？
